README.md

```diff
@@ -1,185 +1,195 @@
 
-#### Inserting and querying data
+    ```sql
+    INSERT INTO conditions
+      VALUES
+        (NOW(), 'office', 70.0, 50.0),
+        (NOW(), 'basement', 66.5, 60.0),
+        (NOW(), 'garage', 77.0, 65.2);
+    ```
 
-Inserting data into the hypertable is done via normal SQL commands:
+- Return the number of entries written to the table conditions in the last 12 hours:
+
+    ```sql
+    SELECT COUNT(*) FROM conditions
+      WHERE time > NOW() - INTERVAL '12 hours';
+    ```
+
+See more:
+
+- [Query data](https://docs.timescale.com/use-timescale/latest/query-data/)
+- [Write data](https://docs.timescale.com/use-timescale/latest/write-data/)
+
+# Time buckets
+
+Time buckets enable you to aggregate data in hypertables by time interval and calculate summary values.
+
+For example, calculate the average daily temperature in a table named `weather_conditions`. The table has a `time` and `temperature` columns:
 
 ```sql
-INSERT INTO conditions(time, location, temperature, humidity)
-  VALUES (NOW(), 'office', 70.0, 50.0);
-
-SELECT * FROM conditions ORDER BY time DESC LIMIT 100;
-
-SELECT time_bucket('15 minutes', time) AS fifteen_min,
-    location, COUNT(*),
-    MAX(temperature) AS max_temp,
-    MAX(humidity) AS max_hum
-  FROM conditions
-  WHERE time > NOW() - interval '3 hours'
-  GROUP BY fifteen_min, location
-  ORDER BY fifteen_min DESC, max_temp DESC;
+SELECT time_bucket('1 day', time) AS bucket,
+  avg(temperature) AS avg_temp
+FROM weather_conditions
+GROUP BY bucket
+ORDER BY bucket ASC;
 ```
 
-In addition, TimescaleDB includes additional functions for time-series
-analysis that are not present in vanilla PostgreSQL. (For example, the `time_bucket` function above.)
+See more:
 
-- [Quick start: Basic operations](https://tsdb.co/GitHubTimescaleBasicOperations)
-- [Reference examples](https://tsdb.co/GitHubTimescaleWriteData)
-- [TimescaleDB API](https://tsdb.co/GitHubTimescaleAPI)
+- [About time buckets](https://docs.timescale.com/use-timescale/latest/time-buckets/about-time-buckets/)
+- [API reference](https://docs.timescale.com/api/latest/hyperfunctions/time_bucket/)
+- [All TimescaleDB features](https://docs.timescale.com/use-timescale/latest/)
+- [Tutorials](https://docs.timescale.com/tutorials/latest/)
 
-### Installation
+# Continuous aggregates
 
-[Timescale Cloud](https://tsdb.co/GitHubTimescale), a fully-managed TimescaleDB in the cloud, is
-available via a free trial. Create a PostgreSQL database in the cloud with TimescaleDB pre-installed
-so you can power your application with TimescaleDB without the management overhead.
+Continuous aggregates are designed to make queries on very large datasets run faster. They use PostgreSQL [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html) to continuously and incrementally refresh a query in the background, so that when you run the query, only the data that has changed needs to be computed, not the entire dataset.
```

Comment:
Actually CAggs don't use Postgres Materialized Views... in fact CAggs is our MatView implementation with steroids (Incremental Refresh, Data Retention, Compression/Columnar Storage) ... but we can´t say that it is a Materialized View because users will expect to see it under MatView sections on their favourite client applications connected to the database but actually they will apear under the View sections cause we expose it to users as a Postgres View.

Reply:
Is there a good approved description anywhere that I could reuse?

Reply:
How does this look ? 

`Continuous aggregates are designed to make queries on very large datasets run faster. They continuously and incrementally refresh a query in the background, so that when you run the query, only the data that has changed needs to be computed, not the entire dataset. Normal PostgreSQL [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html) cannot be incrementally materialized, they have to be rebuilt from scratch every time you want to refresh it. `

Reply:
Added with a minor tweak, thank you